src/MessageContainer.js

```diff
@@ -1,29 +1,50 @@
-import React, { Component, PropTypes } from 'react';
+import React, {Component, PropTypes} from 'react';
+
+import ReactNative, {
+  View,
+  ListView,
+  UIManager
```

Comment:
Nope, I tried to make the content stick to the top when its smaller then the container, it didn't work that well so I removed it.
I will remove the left over reference